server/src/routes/api/v1/placesRouter.js

```diff
@@ -3,6 +3,12 @@ import express from "express"
 
 const placesRouter = new express.Router()
 
-
+placesRouter.get("/", async (req, res) => {
+  try {
+    return res.status(200).json({ body })
+  } catch (error) {
+    return res.status(500).json({ errors: error })
+  }
+})
```

Comment:
hmmm, I dont see any Objection queries here, and I dont know that `body` is defined at all in this route!

Reply:
Alas, placesRouter doesn't get love until the next pull request. But I'm going to leave this as-is for now since it will avoid making any merge conflicts for the next branch.